repos/spack_repo/builtin/packages/blis/package.py

```diff
@@ -83,8 +90,12 @@ def configure_args(self):
         return config_args
 
     def edit(self, spec, prefix):
-        # To ensure auto should always be the last argument for base and derived class
-        config_args = self.configure_args() + ["auto"]
+        # To ensure the target should always be the last argument for base and derived class
+        target = "auto"
+        for _arch, _target in _targets:
+            if self.spec.satisfies(f"target={_target}"):
+                target = _target
```

Comment:
I assume this is to map a Spack name to a Blis name, and you meant:
```suggestion
                target = _arch
```
? Otherwise I don't see where `_arch` is used...